src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -39,15 +43,22 @@ static void OnBackButonBehaviorPropertyChanged(BindableObject bindable, object o
 				SetInheritedBindingContext(newHandlerBehavior, bindable.BindingContext);
 		}
 
-		/// <summary>Bindable property for attached property <c>PresentationMode</c>.</summary>
+		/// <summary>
+		/// Define the small navigation animation that occurs when a page is navigated to with the GoToAsync method.
```

Comment:
It would be good on this one to also indicate that this controls if something is pushed as modal or not. 

The choices here define if something is pushed modally and if it's animated

Reply:
Added.